app/models/user.rb

```diff
@@ -58,18 +58,14 @@ def payer_balance(payer)
     balance.balance_for(payer)
   end
 
-  def total_balance
-    balance.total
+  def total_debt
+    balance.total_debt
   end
```

Comment:
Not that it's wrong per se, but maybe a `delegate :total_debt, to: :balance` would be more elegant?